api/fittings/lisk_cache.js

```diff
@@ -13,9 +13,26 @@
  */
 'use strict';
 
-var modules = require('../../helpers/swagger_module_registry');
 var debug = require('debug')('swagger:lisk:cache');
+var modules = require('../../helpers/swagger_module_registry');
 
+/**
+ * Description of the module.
+ * @module
```

Comment:
Why not follow the pattern from the `api/controllers` PR with a namespace?

Reply:
good idea :)